pose_lookup/plugin.cpp

```diff
@@ -0,0 +1,158 @@
+#include "common/phonebook.hpp"
```

Comment:
This entire file is about the same as pose_predict/plugin.cpp. Why the duplication? Why not inherit from pose_prediction_impl and just override the one or two methods that are necessary? I understand that _impl's are supposed to be internal, so we'll need some refactoring, but it might be worth it given that the code is pretty much identical.

Reply:
Let's talk about this more after 2.2.